src/pages/api/index.tsx

```diff
@@ -0,0 +1,275 @@
+
+    @media (min-width: ${theme.layout.tablet}) {
+        flex-direction: row;
+    }
+
+    .Description {
+        display: flex;
+        flex-direction: column;
+        gap: ${theme.space.space24};
+    }
+
+    .DescriptionLinks {
+        display: flex;
+        gap: ${theme.space.space16};
+
+        a {
+            font-weight: bold;
+            text-wrap: nowrap;
+            max-height: 36px;
+        }
+    }
+
+    .MainSectionContent {
+        display: flex;
+        flex-direction: column;
+        gap: ${theme.space.space16};
+
+        a {
+            text-wrap: nowrap;
+            font-weight: bold;
+            font-size: 14px;
+            max-height: 36px;
+        }
+    }
+`;
+
+const RelatedArticlesWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 2.4rem;
+
+    @media (min-width: ${theme.layout.tablet}) {
+        flex-direction: row;
+    }
+
+    a {
+        width: 100%;
+
+        .ImageWrapper {
+            overflow: hidden;
+        }
+
+        img {
+            transition: transform 120ms;
+        }
+
+        &:hover {
+            img {
+                transform: scale(1.05);
+            }
+        }
+    }
+`;
+
+const ClientCodeWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${theme.space.space24};
+
+    @media (min-width: ${theme.layout.tablet}) {
+        max-width: 50%;
+    }
+`;
+
+const TabTitleWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    gap: ${theme.space.space8};
+`;
+
+const BlogImageWrapper = styled.img`
+    border-radius: 12px;
+    height: 100%;
+`;
+
+export default function Api() {
+    return (
+        <Layout>
+            <UiLibraryWrapper>
+                <HeroWrapper
+                    heading="Apify API"
+                    isCentered
+                    description={<>Apify API provides programmatic access to the <Link to="/">Apify Platform</Link></>}
+                />
+                <SectionWrapper
+                    className={styles.LargerContent}
+                    heading="API reference"
+                    description={<div className="MainSectionContent">
+                        <p>The Apify API allows developers to interact programmatically apps using HTTP requests.
+                        The Apify API is built around <Link to="https://en.wikipedia.org/wiki/REST">REST</Link>.</p>
+                        <p>The API has predictable resource-oriented URLs, returns JSON-encoded responses,
+                        and uses standard HTTP response codes, authentication, and verbs.</p>
+                        <div>
+                            <Button to='/api/v2'>Check API reference</Button>
+                        </div>
+                    </div>}
+                >
+                    <ClientCodeWrapper>
+                        <CodeBlock content={[{ key: 'cURL', label: 'cURL', language: 'bash', code: `
+                # Prepare Actor input
+
```

Comment:
Let's remove the empty lines from the code examples (between the comment and code)